### PR TITLE
[zh-cn]: update the translation of CSS `outline-offset` property

### PR DESCRIPTION
**Pull request by T34-active**
### Description
https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset

### Related issues and pull requests

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/outline-offset/index.md
* Last commit: https://github.com/mdn/content/commit/7a9f9baa25d9a7313bd6c62ef5ef585b28459c58

### CODE REVIEW

File: files/zh-cn/web/css/outline-offset/index.md [hunk 1]
```diff
@@ -1,71 +1,85 @@
...
 title: outline-offset
 slug: Web/CSS/outline-offset
+l10n:
+  sourceCommit: 7a9f9baa25d9a7313bd6c62ef5ef585b28459c58
 ---
 
 {{CSSRef}}
 
-## 概要
+**`outline-offset`** CSS 属性设置了 [outline](/zh-CN/docs/Web/CSS/outline)（轮廓线）与元素边缘或边框之间的间距。
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-offset/index.md:10]:
```suggestion
**`outline-offset`** CSS 属性设置了 [outline](/zh-CN/docs/Web/CSS/outline) 与元素边缘或边框之间的间距。
```

File: files/zh-cn/web/css/outline-offset/index.md [hunk 2]
```diff
@@ -1,71 +1,85 @@
...
 title: outline-offset
 slug: Web/CSS/outline-offset
+l10n:
+  sourceCommit: 7a9f9baa25d9a7313bd6c62ef5ef585b28459c58
 ---
 
 {{CSSRef}}
 
-## 概要
+**`outline-offset`** CSS 属性设置了 [outline](/zh-CN/docs/Web/CSS/outline) 与元素边缘或边框之间的间距。
```
Comment (yin1999) [inline files/zh-cn/web/css/outline-offset/index.md:10]:
```suggestion
**`outline-offset`** CSS 属性设置[轮廓](/zh-CN/docs/Web/CSS/outline)与元素边缘或边框之间的间距。
```

File: files/zh-cn/web/css/outline-offset/index.md [hunk 3]
```diff
@@ -1,71 +1,85 @@
...
+### 值
+
+- `{{cssxref("&lt;length&gt;")}}`
+  - : 元素与其轮廓线之间的间距宽度。负值会将轮廓线置于元素内部。当值为 `0` 时，轮廓线与元素之间没有间距。
+
+## 描述
 
-- `<length>`
-  - : 元素和轮廓间的宽度。详见 {{cssxref("&lt;length&gt;")}} . 负值将轮廓绘制在元素之内。
+轮廓线是绘制在元素周围、边框边缘之外的一条线。元素与其轮廓线之间的空间是透明的。换句话说，它与父元素的背景相同。
```
Comment (yin1999) [inline files/zh-cn/web/css/outline-offset/index.md:36]:
```suggestion
轮廓线是绘制在元素周围、边框边缘之外的一条线。元素与其轮廓线之间的空间是透明的。换句话说，它与父元素具有相同的背景。
```

File: files/zh-cn/web/css/outline-offset/index.md [hunk 4]
```diff
@@ -1,71 +1,85 @@
...
 outline-offset: initial;
+outline-offset: revert;
+outline-offset: revert-layer;
 outline-offset: unset;
 ```
 
-### Values
+### 值
+
+- `{{cssxref("&lt;length&gt;")}}`
```
Comment (yin1999) [inline files/zh-cn/web/css/outline-offset/index.md:31]: 可以去掉，英文文档的也可删去

```suggestion
- {{cssxref("&lt;length&gt;")}}
```

Comment (T34-active) [reply files/zh-cn/web/css/outline-offset/index.md:31]: https://github.com/mdn/content/pull/33676

File: files/zh-cn/web/css/outline-offset/index.md [hunk 5]
```diff
@@ -1,71 +1,85 @@
 ---
 title: outline-offset
 slug: Web/CSS/outline-offset
+l10n:
+  sourceCommit: 7a9f9baa25d9a7313bd6c62ef5ef585b28459c58
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-offset/index.md:5]:
```suggestion
  sourceCommit: cc4592258b952f452e8715b67b14ce3534ccfa69
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**